xtest/test_tdfs.py

```diff
@@ -146,16 +155,42 @@ def test_tdf_spec_target_422(
         encrypt_sdk,
         "ztdf",
         tmp_dir,
+        scenario="target-422",
         target_mode="4.2.2",
     )
     assert os.path.isfile(ct_file)
 
+    manifest = tdfs.manifest(ct_file)
+    assert manifest.payload.isEncrypted
+
+    looks_like_422(manifest)
+
     fname = os.path.basename(ct_file).split(".")[0]
     rt_file = f"{tmp_dir}test-{fname}.untdf"
     decrypt_sdk.decrypt(ct_file, rt_file, "ztdf")
     assert filecmp.cmp(pt_file, rt_file)
 
 
+def looks_like_422(manifest: tdfs.Manifest):
+    assert manifest.schemaVersion is None
+
+    ii = manifest.encryptionInformation.integrityInformation
+    # in 4.2.2, the root sig is hex encoded before base 64 encoding, and is twice the length
+    assert len(ii.rootSignature.sig) == 88
```

Comment:
would it be easier to just check if hex decode succeeds? then we wouldnt have to worry if we change what were encrypting